src/_layouts/docs.html

```diff
@@ -2,89 +2,80 @@
 layout: default
 ---
 {% assign page_href = page.url | slice: 1, page.url.size %}
-<div class="wrapper wrapper--docs u-marginBottom1_5">
-  <nav class="subnav" aria-label="Most Popular">
+<div class="Docs-wrapper">
+  <header class="Docs-header">
+    <nav class="Docs-nav">
+      <ul class="Docs-nav-list">
+        {% for link in site.data.docs.top %}
+          <li>
+            <a class="Docs-nav-link {% if page_href == link.href %}is-active{% endif %}" href="/{{link.href | escape}}">{{link.title | escape}}</a>
+          </li>
+        {% endfor %}
+      </ul>
+    </nav>
     <form class="js-only" action="/search" method="GET">
-      <div class="formItem">
-        <input type="text" name="q" id="q" placeholder="Search" aria-label="Search Documentation" />
```

Comment:
I did a listen of this page with Voiceover, and I noticed that it reads the placeholder "Search"...so the additional label of "Search Documentation" seemed redundant and added slightly to the read time.